[2022-09-18 03:00:47,293] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.python_task_1 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:00:47,442] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: first_py_dag.python_task_1 2022-09-17T03:00:00+00:00 [queued]>
[2022-09-18 03:00:47,447] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:00:47,453] {taskinstance.py:1018} INFO - Starting attempt 1 of 3
[2022-09-18 03:00:47,467] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-09-18 03:00:47,531] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): python_task_1> on 2022-09-17T03:00:00+00:00
[2022-09-18 03:00:47,580] {standard_task_runner.py:51} INFO - Started process 26742 to run task
[2022-09-18 03:00:47,603] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'first_py_dag', 'python_task_1', '2022-09-17T03:00:00+00:00', '--job-id', '247', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_operator.py', '--cfg-path', '/tmp/tmpzy7d_31a']
[2022-09-18 03:00:47,613] {standard_task_runner.py:76} INFO - Job 247: Subtask python_task_1
[2022-09-18 03:00:47,760] {logging_mixin.py:103} INFO - Running <TaskInstance: first_py_dag.python_task_1 2022-09-17T03:00:00+00:00 [running]> on host 27377494ab8b
[2022-09-18 03:00:47,909] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Dave
AIRFLOW_CTX_DAG_ID=first_py_dag
AIRFLOW_CTX_TASK_ID=python_task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-09-17T03:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-17T03:00:00+00:00
[2022-09-18 03:00:47,948] {logging_mixin.py:103} INFO - hELLO Layo
[2022-09-18 03:00:47,950] {logging_mixin.py:103} INFO - 
[2022-09-18 03:00:47,951] {python.py:118} INFO - Done. Returned value was: None
[2022-09-18 03:00:48,006] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=first_py_dag, task_id=python_task_1, execution_date=20220917T030000, start_date=20220918T030047, end_date=20220918T030048
[2022-09-18 03:00:48,476] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-09-18 03:00:48,528] {local_task_job.py:118} INFO - Task exited with return code 0
